src/database/migrations/20210727095258-create-subject.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Subjects', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      subjectName: {
+        allowNull: false,
+        type: Sequelize.STRING,
```

Comment:
Название предмета уникальное
```suggestion
      subjectName: {
        allowNull: false,
        unique: true,
        type: Sequelize.STRING,
```

Reply:
@BauyrjanSeidahmet Переправил